test/deprecations.js

```diff
@@ -0,0 +1,50 @@
+var morgan = require('..')
+var assert = require('assert')
+
+describe('morgan() deprecations', function () {
+  var originalConsoleError
+  var errors = []
+  function getErrors () {
+    return errors.map(function (e) { return removeANSI(e.toString('utf8')) })
+  }
+  before(function () {
+    originalConsoleError = process.stderr.write
+
+    process.stderr.write = function write (chunk, encoding) {
+      /* eslint node/no-deprecated-api: 0 */ // this is actually trying to be compat with older node
+      errors.push(Buffer.from ? Buffer.from(chunk, encoding) : new Buffer(chunk, encoding))
+    }
+  })
+  after(function () {
+    if (originalConsoleError) {
+      process.stderr.write = originalConsoleError
+    }
+  })
+  afterEach(function () {
+    errors = []
+  })
+
+  it('should tell you to use morgan "combined" if format arg is an object', function () {
```

Comment:
I think there is a typo somewhere as the description doesn't match the actual test. From looking at it, I think the description is what just needs the update 👍